models/schema_template_vrf.go

```diff
@@ -11,15 +11,16 @@ func NewSchemaTemplateVrf(ops, path, Name, displayName, ipDataPlaneLearning stri
 
 	if ops != "remove" {
```

Comment:
Do we still need this, as i understood new delete PATCH method is implemented? Or keeping intended?

Reply:
yes keeping is intended.